### PR TITLE
Add note for sending email via /usr/sbin/sendmail

### PR DESCRIPTION
**Pull request by mrkschan**

### CODE REVIEW

**Comment by pjcdawkins** [comment]
How does this information help the user?

To me it seems like a mere implementation detail.

**Comment by mrkschan** [comment]
Yes, for PHP container. You don't need to know that. What about non-PHP container?

**Comment by Crell** [comment]
This change doesn't really tell Node or Python users how they should then send email, does it?  We should generalize beyond PHP, but this doesn't provide much help for non-PHP users.

**Comment by pjcdawkins** [comment]
May I correct:

>`/usr/sbin/sendmail` -> `sendmail`

> `mail` facility -> `mail()` facility

**Comment by mrkschan** [comment]
Updated, @Crell @pjcdawkins. Note, `/usr/sbin/` is not included in `$PATH` by default and thus we need `/usr/sbin/sendmail`.

**Comment by mrkschan** [comment]
Regarding `mail` vs `mail()`, i have no idea in PHP :) Please let me know which is the official one. Also, shall we correct the `file spool` facility as well?

File: reference/email.md [hunk 1]
```diff
@@ -1,25 +1,28 @@
...
-Emails from Platform.sh are sent via a SendGrid-based SMTP proxy. Each
-Platform.sh project is provisioned as a SendGrid sub-account. Note that SendGrid
-sub-accounts are capped at 12k emails per month. If you need to adopt SPF for
-improving email deliverability, please read
-[SPF setup guidelines on SendGrid](https://sendgrid.com/docs/Glossary/spf.html).
+## Sending email in PHP
 
-When you send email you should use the `mail` facility. There is no SMTP 
-server running on localhost. For example in Symfony, if you use the default `SwiftMailer`,
+When you send email, you should use the built-in `mail` facility in PHP.
```
Comment (Crell) [inline reference/email.md]: Let's say "the build-in `mail()` function in PHP".  No room for confusion that way.

File: reference/email.md [hunk 2]
```diff
@@ -1,25 +1,28 @@
...
+PHP runtime is configured to send email via the assigned SendGrid sub-account.
```
Comment (Crell) [inline reference/email.md]: Needs a "The" at the beginning.  And this doesn't need to be a new paragraph now.

**Comment by Crell** [comment]
The file spool is a SwiftMailer thing, I think?  Not something PHP-specific, so I think that's fine as is.

File: reference/email.md (hunk 2 above)
Comment (mrkschan) [inline reference/email.md]: New paragraph? Do you mean the mention of Symfony in the next paragraph?

**Pull request merged**